src/fulfiller.js

```diff
@@ -118,6 +118,14 @@ class Fulfiller {
     return this._links && this._links.fulfillerLogo ? this._links.fulfillerLogo.href : null;
   }
 
+  getFulfillerContactsUrl() {
```

Comment:
This just exposes the link and the user of the package still has to know how to call the endpoint (specifying the header and stuff). 

The original intent to add contacts support, was to expose contacts in similar way the package works with fulfiller. You call something like `getContacts` and you get an array of plain objects with contact details and you don't need to know how to retrieve them, what's the technology there etc.

Reply:
Yes, That was my initial purpose, but I saw that there was a link pointing to the fulfiller contacts that was not exposed and I though that that ended the problem. But the initial point was good so I think I will finally implement that function.